Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Проект Студентов группы ЭУ-201\r\nдоска объявлений с игровыми предметами\r\n\r\nПоллак Игорь harry1pacman\r\nЯщенко Татьяна TatianaYash\r\nУсков Артем uskovdd\r\nБелорукова Юля juliawhitehand\r\n\r\n\r\nТехническое задание: [tech.md](tech.md)\r\nСтруктура базы данных: [BD_structure.md](BD_structure.md)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 4a03149510b21b1748989b39a13f34de8a2d36a4)
+++ b/README.md	(date 1748352479657)
@@ -3,7 +3,7 @@
 
 Поллак Игорь harry1pacman
 Ященко Татьяна TatianaYash
-Усков Артем uskovdd
+Усков Артем uskovdd 
 Белорукова Юля juliawhitehand
 
 
