Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Проект Студентов группы ЭУ-201\r\nдоска объявлений с игровыми предметами\r\n\r\nПоллак Игорь harry1pacman\r\nЯщенко Татьяна TatianaYash\r\nУсков Артем uskovdd\r\nБелорукова Юля juliawhitehand\r\n\r\n\r\nТехническое задание: [TECH.md](TECH.md)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision c7deec1bee9f82f9c8b70afc48e0a6ec0a2898ba)
+++ b/README.md	(date 1747906486106)
@@ -3,7 +3,7 @@
 
 Поллак Игорь harry1pacman
 Ященко Татьяна TatianaYash
-Усков Артем uskovdd
+Усков Артем uskovdd 
 Белорукова Юля juliawhitehand
 
 
